RFIDForIoT/CS/README.md

```diff
@@ -0,0 +1,95 @@
+---
+page_type: sample
+urlFragment: RFIDForIoT
+languages:
+  - csharp
+products:
+  - Windows 10
+  - Windows IoT 
+  - Windows 10 IoT Enterprise 
+description: Sample to show how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans
+---
+
+# RFID scanner with Windows 10 IoT Enterprise
+
+In this sample, we will demonstrate how to read the RFID Tag from MFRC522 Scanner and Beep the Buzzer when the card Scans.
+Keep in mind that the GPIO APIs are only available on Windows 10 IoT Enterprise boards, so this sample cannot run on your desktop.
+
+## Step 1: Load the project in Visual Studio
+
+## Step 2: Connect the MFRC522 to your Windows 10 IoT Enterprise device
+
+You'll need a few components:
+
+* MFRC522 scanner
+* a Piezo Buzzer (If you want a beep when a card scans)
+* a breadboard and a couple of connetor wires
+
+### For UP Board
+
+1. Connect RFID SDA to Pin 24
+2. Connect RFID SCK to Pin 23
+3. Connect RFID MOSI to Pin 19
+4. Connect RFID MISO to Pin 21
+5. Connect RFID GND to Pin 6
+6. Connect RFID RST to Pin 38
+7. Connect RFID 3.3V to Pin 1 (For some higher frequency cards this might need 5V)
+8. Connect Piezo Buzzer '+' to Pin 32
+9. Connect Piezzo Buzzer '-' to Pin 6

```

Comment:
Piezzo is spelled incorrectly, should be Piezo.

Reply:
Updated the Changes